notebooks/metrics_demo.ipynb

```diff
@@ -18,11 +24,6 @@
     "import cuml\n",
     "import sklearn\n",
     "\n",
-    "from cuml.cluster import KMeans\n",
-    "from cuml.ensemble import RandomForestRegressor as curfr\n",
-    "from cuml.ensemble import RandomForestClassifier as curfc\n",
-    "from cuml.manifold.umap import UMAP as cuUMAP\n",
-    "\n",
     "from sklearn import manifold\n",
     "from sklearn.datasets import make_classification, make_regression\n",
```

Comment:
We are going to want to use the cuml dataset generators for this end-to-end example